client/src/components/AddGames/index.js

```diff
@@ -1,22 +1,23 @@
 import React from "react";
-import M from "materialize-css";
```

Comment:
still need this as well to get anything from materialize to work properly

Reply:
this is already in there, you must be seeing an old commit. this is only needed so that the dropdowns work when navigating from any page